charts/pcns/values.yaml

```diff
@@ -219,3 +219,27 @@ patroni:
 
   # persistentVolume:
   #   enabled: false
+
+peachSync:
+  enabled: false
+
+  # -- Cron schedule for PEACH sync
+  schedule: "5 3 * * *"
+
+  # -- Suspend subsequent runs of the CronJob without deleting it
+  suspend: false
+
+  # -- How long the controller can miss the schedule before it gives up starting the job
+  startingDeadlineSeconds: 600
+  concurrencyPolicy: Forbid
+
+  successfulJobsHistoryLimit: 2
+  failedJobsHistoryLimit: 5
+
+  backoffLimit: 1
+  activeDeadlineSeconds: 1800
+  terminationGracePeriodSeconds: 30
+
+  resources:
+    requests: { cpu: "200m", memory: "512Mi" }
+    limits:   { cpu: "1",    memory: "2Gi" }
```

Comment:
Two things:
- This should be defined in yaml format, not as a JSON object insert
- Are these values what are necessary for the job to execute? The requests and limits seem a bit on the high side at a quick glance.